Source/ActorPool/Private/ActorPoolSettings.cpp

```diff
@@ -1,15 +1,18 @@
 #include "ActorPoolSettings.h"
 
-FAPPooledActorAcquireFromPoolSettings::FAPPooledActorAcquireFromPoolSettings():
+FAPPooledActorAcquireFromPoolSettings::FAPPooledActorAcquireFromPoolSettings() :
     bShowActor( true ),
     bEnableCollision( true ),
     bDisableNetDormancy( true ),
     NetDormancy( ENetDormancy::DORM_Awake )
 {}
 
-FActorPoolInfos::FActorPoolInfos():
+FActorPoolInfos::FActorPoolInfos() :
     Count( 0 ),
-    bAllowNewInstancesWhenPoolIsEmpty( true )
+    bAllowNewInstancesWhenPoolIsEmpty( true ),

```

Comment:
there's no point in having one bool to allow to create new instances and one other to use a circular buffer. It's either one of the other
I would use a policy enum

Reply:
Done.